db/helpers/common.js

```diff
@@ -17,6 +17,7 @@ const pub = require('../../cache/redisCache').client.pubPerspective;
 const dbconf = require('../../config').db;
 const channelName = require('../../config').redis.perspectiveChannelName;
 const revalidator = require('revalidator');
+const joi = require('joi');
```

Comment:
what about removing references to the revalidator module, and removing it from package.json too?